packages/edit-site/src/components/editor/index.js

```diff
@@ -179,8 +180,6 @@ function Editor( { initialSettings } ) {
 	return (
 		<>
 			<URLQueryController />
-			<FullscreenMode isActive />
-			<UnsavedChangesWarning />
```

Comment:
Do you know if moving these two components has any impact?
Other than that, this is looking good.

Reply:
I tested, and I wasn't able to spot any.

The `is-full-screen` class is applied to the body, and an "unsaved changes warning" is triggered if I try to reload the page.